src/utilities/indexer/fragments.ts

```diff
@@ -0,0 +1,36 @@
+// GraphQL provides reusable units called fragments.
+// Fragments let you construct sets of fields, and then include them in queries where needed.
+
+export const wholeBlock = `
+fragment wholeBlock on Block {
+  id
+  hash
+  timeStamp
+}`;
+
+/** If used, 'wholeBlock' also needs to be included. */
```

Comment:
Maybe...

Reply:
I think that in that case it would be better replace the use of "wholeBlock" inside of "wholeAttestation" and just write the whole thing out. 

But this then also serves as a showcase on how to use the fragments itself.

Reply:
If you include a fragment twice on a request, it leads to an error. 

It is as possible as possible that someone tries to include both than they forget the dependee.